Library/Homebrew/formula_auditor.rb

```diff
@@ -315,7 +315,29 @@ def audit_deps
 
     def audit_conflicts
       formula.conflicts.each do |c|
-        Formulary.factory(c.name)
+        conflicting_formula = Formulary.factory(c.name)
+        problem "Formula should not conflict with itself" if formula == conflicting_formula
+
+        next unless @core_tap
+
+        if CoreTap.instance.formula_renames.key?(c.name) || Formula.aliases.include?(c.name)
```

Comment:
```suggestion
        if formula.tap.formula_renames.key?(c.name) || formula.tap.aliases.include?(c.name)
```

Does this still work? I think it improves support for non-core formulae a bit

Reply:
Fixed! It took some time for me to figure out how to update tests for this